Add documentation for TeamsMeetingBrandingPolicy cmdlets (#10657)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Set-CsTeamsMeetingBrandingPolicy.md

---
external help file: Microsoft.Teams.Policy.Administration.Cmdlets.Core.dll-Help.xml
Module Name: MicrosoftTeams
online version: https://learn.microsoft.com/powershell/module/skype/set-csteamsmeetingbrandingpolicy
schema: 2.0.0
title: Set-CsTeamsMeetingBrandingPolicy
author: szymonkatraMSFT
ms.author: szymonkatra
ms.reviewer:
manager: stanlythomas
applicable: Skype for Business Online
---

# Set-CsTeamsMeetingBrandingPolicy

## SYNOPSIS

> The CsTeamsMeetingBrandingPolicy cmdlets enable administrators to control the appearance in meetings by defining custom background, logo and colors.

Reviewer: ```suggestion
The CsTeamsMeetingBrandingPolicy cmdlet enables administrators to control the appearance in meetings by defining custom backgrounds, logos, and colors.
```

 [-MeetingBackgroundImages <PSListModifier>]
 [-MeetingBrandingThemes <PSListModifier>]
 [-DefaultTheme <String>] [-EnableMeetingOptionsThemeOverride <Boolean>]
 [-EnableMeetingBackgroundImages <Boolean>] [-Identity] <String> [-Force] [-WhatIf] [-Confirm]
 [<CommonParameters>]
```

## DESCRIPTION
The Set-CsTeamsMeetingBrandingPolicy cmdlet allows administrators to update existing meeting branding policies.
However, it cannot be used to upload the images. If you want to upload the images, please use Teams Admin Center.

## EXAMPLES

### Example 1
```powershell
PS C:\> Get-CsTeamsMeetingBrandingPolicy
PS C:\> $brandingPolicy = Get-CsTeamsMeetingBrandingPolicy -Identity "demo branding"
PS C:\> $brandingPolicy.MeetingBrandingThemes[0].BrandAccentColor = "#FF0000"
PS C:\> Set-CsTeamsMeetingBrandingPolicy -Identity "demo branding" -MeetingBrandingThemes $brandingPolicy.MeetingBrandingThemes
```

In the example shown above, the commands will change brand accent color of theme inside the `demo branding` meeting branding policy to `#FF0000`. 

## PARAMETERS

### -MeetingBackgroundImages
*This parameter is reserved for Microsoft internal use only.*
List of meeting background images.

> It is not possible to add/remove background images using cmdlets. Please use Teams Admin Center (TAC) for that purpose.

Reviewer: ```suggestion
It is not possible to add or remove background images using cmdlets. You should use Teams Admin Center for that purpose.
```

```yaml
Type: PSListModifier
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -MeetingBrandingThemes

> List of meeting branding themes. You can alter list returned by `Get-CsTeamsMeetingBrandingPolicy` cmdlet and pass it to this parameter. (see example 1).
> It is not possible to add new/remove meeting branding themes using cmdlets. Please use Teams Admin Center (TAC) for that purpose.

Reviewer: ```suggestion
List of meeting branding themes. You can alter the list returned by the `Get-CsTeamsMeetingBrandingPolicy` cmdlet and pass it to this parameter.
It is not possible to add or remove meeting branding themes using cmdlets. You should use Teams Admin Center for that purpose.
```

```yaml
Type: PSListModifier
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -DefaultTheme
*This parameter is reserved for Microsoft internal use only.*
Identity of default meeting theme.

```yaml
Type: String
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -EnableMeetingBackgroundImages

> Enable custom meeting backgrounds.

Reviewer: ```suggestion
Enables custom meeting backgrounds.
```

```yaml
Type: SwitchParameter
Parameter Sets: (All)
Aliases: cf

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -Force
Suppresses any confirmation prompts that would otherwise be displayed before making changes.

```yaml
Type: SwitchParameter
Parameter Sets: (All)
Aliases: 

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -WhatIf
Describes what would happen if you executed the command without actually executing the command.

```yaml
Type: SwitchParameter
Parameter Sets: (All)
Aliases: wi

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### CommonParameters
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).

## NOTES

Available in Teams PowerShell Module 4.9.3 and later.

## RELATED LINKS

> [Get-CsTeamsMeetingBrandingPolicy](Get-CsTeamsMeetingBrandingPolicy.md)
> [Grant-CsTeamsMeetingBrandingPolicy](Grant-CsTeamsMeetingBrandingPolicy.md)
> [New-CsTeamsMeetingBrandingPolicy](New-CsTeamsMeetingBrandingPolicy.md)
> [Remove-CsTeamsMeetingBrandingPolicy](Remove-CsTeamsMeetingBrandingPolicy.md)
> [Set-CsTeamsMeetingBrandingPolicy](Set-CsTeamsMeetingBrandingPolicy.md)

Reviewer 2: ```suggestion
[Get-CsTeamsMeetingBrandingPolicy](Get-CsTeamsMeetingBrandingPolicy.md)

[Grant-CsTeamsMeetingBrandingPolicy](Grant-CsTeamsMeetingBrandingPolicy.md)

[New-CsTeamsMeetingBrandingPolicy](New-CsTeamsMeetingBrandingPolicy.md)

[Remove-CsTeamsMeetingBrandingPolicy](Remove-CsTeamsMeetingBrandingPolicy.md)

[Set-CsTeamsMeetingBrandingPolicy](Set-CsTeamsMeetingBrandingPolicy.md)
```
